webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -187,7 +187,12 @@ def bugzilla_submit_url(report, parsed_dump, crashing_thread, bug_product):
         "comment": comment,
     }
 
-    # some special keys have to be truncated to make Bugzilla happy
+    # If this crash report has DOMFissionEnabled=1, then we prepend "[Fission]"
+    # to the summary. This is probably temporary. Bug #1659175.
+    if raw_crash.get("DOMFissionEnabled"):
+        kwargs["short_desc"] = "[Fission] " + kwargs["short_desc"]
```

Comment:
We originally wanted to set the `cf_fission_milestone` to `?`, but Bugzilla doesn't support setting project tracker flags using the querystring when creating a bug.